README.md

```diff
@@ -24,38 +24,42 @@ Right now here are the differences:
  - FiniteDiff.jl is carefully optimized to minimize allocations
  - FiniteDiff.jl supports coloring vectors for efficient calculation of sparse Jacobians
 
-## Examples
+
+#### FDM.jl
+This package was formerly called FDM.jl. We recommend users of FDM.jl [update to FiniteDifferences.jl](https://github.com/JuliaDiff/FiniteDifferences.jl/issues/37).
+
+
+## Example: Scalar Derivatives
 
 Compute the first derivative of `sin` with a 5th order central method:
 
 ```julia
 julia> central_fdm(5, 1)(sin, 1) - cos(1)
--1.247890679678676e-13
+-2.4313884239290928e-14
 ```
+
 Compute the second derivative of `sin` with a 5th order central method:
 
 ```julia
 julia> central_fdm(5, 2)(sin, 1) + sin(1)
-9.747314066999024e-12
+-4.367495254342657e-11
```

Comment:
Should I be worried about this loss of accuracy?
I guess not because the first order got more accurate

Reply:
I'm not sure that this loss of accuracy is significant. The whole step size estimation procedure relies on a lot of assumptions and approximations, so a change in accuracy like this could easily be due to another, unrelated change.